contracts/implementations/abstracts/RMRKAbstractEquippableImpl.sol

```diff
@@ -20,6 +25,12 @@ abstract contract RMRKAbstractEquippableImpl is
     uint256 private _totalAssets;
     string private _tokenURI;
 
+    /**
+     * @notice A hook to be called prior to minting tokens.
```

Comment:
Same note about not being meant to be a hook, probably in other places but will stop reporting.